ui/src/pages/datasetUpload/index.vue

```diff
@@ -33,35 +33,51 @@
           color="success"
```

Comment:
Changes:
- Show uploaded dataset's type
- For user role, instead of showing a link to the uploaded dataset, only show the dataset name.